docs/TestingFabricProxyModel.md

```diff
@@ -4,7 +4,15 @@
 
 2.  Set the environment variable ``TCF_HOME`` to the Avalon root directory
 
-3.  Start the Fabric network with 2 organizations, 4 peers and 3 orderers
+3.  If the host machine is behind any firewall/proxy, you need to define the
+    following parameters in /etc/environment file:
+    http_proxy=<http-proxy-url>:<port>
+    https_proxy=<https-proxy-url>:<port>
+    no_proxy=localhost,127.0.0.1,orderer3.example.com,orderer2.example.com,
+             orderer1.example.com,peer2.org1.example.com,peer1.org1.example.com,
+             peer2.org0.example.com,peer1.org0.example.com
```

Comment:
Manual approach to set no_proxy doesn't look scalable w.r.t ease of setup. Can we set these values in docker-compose file or in fabric startup script. Ideally I would like to set no_proxy only while fabric is running than set it at system level.

Reply:
Ideally, proxies should not be hard-coded in source code. This has to be set by the user in his environment. I see no-proxy has been hard-coded in scripts/start_fabric.sh for fixed number of orderers, peers and organisations. I'm going to clean this up and this has to be set by user according to the documentation provided.

Reply:
I'm not certain why should user manually set no_proxy for fabric specific configuration. I can understand setting proxies in /etc/environment manually for http_proxy or https_proxy which is company specific setting. I would let fabric startup script set it based on fabric network configuration rather than manually setting it. @litong01 Please let us know if you have any suggestions for where to set no_proxy for fabric configuration